openmc/mgxs_library.py

```diff
@@ -308,15 +364,12 @@ def name(self, name):
     @energy_groups.setter
     def energy_groups(self, energy_groups):
         # Check validity of energy_groups
-        check_type("energy_groups", energy_groups, EnergyGroups)
+        check_type('energy_groups', energy_groups, openmc.mgxs.EnergyGroups)
 
-        # Check that there is one or more groups
-        if ((energy_groups.num_groups is None) or
-            (energy_groups.num_groups < 1)):
-
-            msg = 'energy_groups object incorrectly initialized.'
+        if energy_group.group_edges is None:
+            msg = 'Unable to assign an EnergyGroups object ' + \
```

Comment:
string addition is implicit when separated only be whitespace. Thus, you can remove the +